TestControllers/AzureController.psm1

```diff
@@ -139,26 +150,23 @@ Class AzureController : TestController
 			$azureConfig.Subscription.ARMStorageAccount = $this.StorageAccount.Trim()
 			Write-LogInfo "Selecting custom storage account : $($azureConfig.Subscription.ARMStorageAccount) as per your test region."
 		}
-		else
-		{
+		else { # else means $this.StorageAccount is empty, or $this.StorageAccount is like 'Auto_Complete_RG=Xxx'
 			$azureConfig.Subscription.ARMStorageAccount = $RegionAndStorageMap.AllRegions.$($this.TestLocation).StandardStorage
 			Write-LogInfo "Auto selecting storage account : $($azureConfig.Subscription.ARMStorageAccount) as per your test region."
 		}
 
-		if( $this.ResultDBTable )
-		{
+		if ($this.ResultDBTable) {
 			$azureConfig.ResultsDatabase.dbtable = ($this.ResultDBTable).Trim()
 			Write-LogInfo "ResultDBTable : $($this.ResultDBTable) added to GlobalConfig.Global.HyperV.ResultsDatabase.dbtable"
 		}
-		if( $this.ResultDBTestTag )
-		{
+		if ($this.ResultDBTestTag) {
 			$azureConfig.ResultsDatabase.testTag = ($this.ResultDBTestTag).Trim()
 			Write-LogInfo "ResultDBTestTag: $($this.ResultDBTestTag) added to GlobalConfig.Global.HyperV.ResultsDatabase.testTag"
 		}
 
 		Write-LogInfo "------------------------------------------------------------------"
 
-		$SelectedSubscription = Select-AzSubscription -SubscriptionId $azureConfig.Subscription.SubscriptionID
+		$SelectedSubscription = Set-AzContext -SubscriptionId $azureConfig.Subscription.SubscriptionID
```

Comment:
why is it changed? Select-AzSubscription is still available.

Reply:
Because Set-AzContext is the normal and recommended one from Az module, instead of an alias.